templates/blog.html

```diff
@@ -1,5 +1,25 @@
 {% extends 'base.html' %}
 {% block head %}
+<!-- Open Graph Tags -->
+<meta property="og:type" content="article" />
+<meta property="og:title" content="{{ article.title|escape }}" />
+<meta property="og:site_name" content="{{ planet.name|escape }}" />
+<meta property="og:description" content="{{ article.content }}" />
+<meta property="article:published_time" content="{{ article.created }}" />
```

Comment:
It seems to require a specific format here:

https://stackoverflow.com/questions/25125612/articlepublished-time-date-format

Reply:
Added a new date formatter for the specific date format of `article:published_time` https://github.com/Planetable/Planet/commit/553e99c97123d7fc8f683741616c15f9e44d634e